src/components/ont/runs/ONTFlowcell.vue

```diff
@@ -66,31 +76,48 @@ export default {
   },
   data() {
     return {
-      barcodeState: null,
+      /**Represents whether the barcode is valid (=true) , invalid(=false) or empty(=null) */
+      barcodeState: true,
     }
   },
   computed: {
-    flowcellIdValidation() {
+    /**Returns an error message if invalid otherwise an empty string */
+    flowcellIdValidationError() {
+      // 3 letters followed by at least 3 numbers
       if (this.flowcellId) {
-        // 3 letters followed by at least 3 numbers
-        return !!this.flowcellId.match(/^[a-zA-Z]{3}\d{3,}$/)
-      } else {
-        return null
+        return this.flowcellId.match(/^[a-zA-Z]{3}\d{3,}$/)
+          ? ''
+          : 'Enter at valid Flowcell ID (3 letters then at least 3 numbers)'
       }
+      return ''
+    },
+    /**Returns a message if invalid otherwise empty */
+    barcodeValidationError() {
+      return this.barcode ? (this.barcodeState ? '' : 'Enter a valid Pool Library barcode') : ''
     },
+    /** Is the flowcellId empty or not */
+    isFlowIdExists() {
```

Comment:
Should this be isFlowcellIdExists?

Reply:
Yes, sure , that make it more clear . Changed now